tests/mealpy_test.py

```diff
@@ -0,0 +1,116 @@
+from unittest import mock
+
+import pytest
+import requests
+import responses
+
+from mealpy import mealpy
+
+
+@pytest.fixture(autouse=True)
+def mock_responses():
+    with responses.RequestsMock() as _responses:
+        yield _responses
+
+
+def test_get_city(mock_responses):
+    response = {
+        'result': [
+            {
+                'id': 'mock_id1',
+                'objectId': 'mock_objectId1',
+                'state': 'CA',
+                'name': 'San Francisco',
+                'city_code': 'SFO',
+                'latitude': 'mock_latitude',
```

Comment:
How about pulling out these mock responses as a test fixture? It looks like you are reusing them (at least partially) in `test_get_city_not_found`.

Reply:
I'm holding off on turning them to fixtures. Depending on how the other tests look, it might or might not end up being heavily reused.

I'm trying to strike a balance between documenting the full response one can expect (as test data for implementing new features) and over-specifying. Which is why `test_get_city_not_found` has a stripped down response because it doesn't care about all the other attributes.

Reply:
I should further add that from my experience, unit tests are easier to grok when they're dumb. Unlike systems, where composition is favoured for reuse, tests can start to have too much contextual overhead (it's a system of its own!) and then you can start introducing bugs in testing code. And who tests the tests...

I'm trying to preserve a full and accurate approximation of the "happy day" response, so it can be used as reference when developing and debuggin. But in edge case tests, I don't want to be weighed down by heavy weight objects.